cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -275,32 +290,32 @@ describe('Script Builder', () => {
         cy.getByTestID('flux-sync--toggle').should('not.have.class', 'active')
         cy.log('select empty dataset')
         cy.getByTestID('flux-editor').monacoType(`{selectall}{enter}
-            from(bucket: "defbuck3") |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
-          `)
+          from(bucket: "defbuck3") |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+        `)
         cy.getByTestID('flux-editor').contains('defbuck3')
 
         runTest(0, 0, false)
       })
 
       it('will return 1 table, for a dataset with only 1 table', () => {
         cy.log('select dataset with 1 table')
-        selectBucket('defbuck4')
-        cy.getByTestID('flux-editor').contains(`from(bucket: "defbuck4")`, {
-          timeout: APPROXIMATE_EDITOR_SET_VALUE_DELAY,
-        })
-        cy.getByTestID('data-explorer--csv-download').should('be.disabled')
-        selectMeasurement('ndbc_1table')
-        cy.getByTestID('flux-editor').contains(
-          `|> filter(fn: (r) => r._measurement == "ndbc_1table")`
-        )
+        cy.getByTestID('csv-download-button').should('be.disabled')
+        cy.getByTestID('flux-editor').monacoType(`{selectall}{enter}
+          from(bucket: "defbuck4") |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+          |> filter(fn: (r) => r._measurement == "ndbc_1table")
+        `)
```

Comment:
Instead of using flux lsp composition, I just type the query into the flux-editor since this test case is mainly testing for the csv download, instead of flux lsp composition

Reply:
good call.